hare/builder.go

```diff
@@ -71,13 +63,11 @@ type innerMessage struct {
 
 // Bytes returns the message as bytes.
 func (im *innerMessage) Bytes() []byte {
-	var w bytes.Buffer
-	_, err := xdr.Marshal(&w, im)
+	buf, err := types.InterfaceToBytes(im)
 	if err != nil {
-		log.Panic("could not marshal InnerMsg before send")
+		panic("could not marshal InnerMsg before send")
```

Comment:
log.panic wraps panic in json, panic outputs the error to err stream, maybe it's best to stay with log.Panic

Reply:
added it back. i just think that if we created struct ourselves encoding shouldn't fail when running a node, and therefore panic is sufficient, but we can keep log.Panic as well